src/app/screens/ledger/verifyLedgerAccountAddress/index.tsx

```diff
@@ -284,6 +300,22 @@ function VerifyLedger(): JSX.Element {
     setCurrentStepIndex(0);
   };
 
+  if (mismatch) {
+    return (
+      <Container>
+        <FullScreenHeader />
+        <LedgerFailViewContainer>
+          <LedgerFailView title={t('TITLE_FAILED')} text={t('ADDRESS_MISMATCH')} />
+          <LedgerFailButtonsContainer>
+            <ActionButtonContainer>
+              <ActionButton onPress={handleWindowClose} text={t('CLOSE_BUTTON')} />
+            </ActionButtonContainer>
+          </LedgerFailButtonsContainer>
+        </LedgerFailViewContainer>
+      </Container>
+    );
+  }
+
```

Comment:
@dhriaznov seems like there's no reason this needs to be on the verify ledger screen, since you're redirecting to it anyway?

let's keep in mind for the future, in case you want a generic error screen to be a catch all from any point on a ledger flow